src/modules/inspect-correlation/inspect-correlation.scss

```
@@ -50,6 +50,14 @@
   flex-shrink: 0;
 }
 
+.inspect-correlation__right-panel__resize-div {
```

Don't go deeper than one element in BEM. This could lead to very unreadable names soon.
Better:

```suggestion
.right-panel__resize-div {
```
or
```suggestion
.inspect-correlation__right-resize-div {
```